# -*- coding:utf-8 -*-
# $ catkin_make -DCATKIN_WHITELIST_PACKAGES="copley"
# 用于控制钩刺机构和推拉机构的运动
# Header
Header header
# Switch
int8 adjust_left
int8 adjust_right
bool enc_wing
bool enc_sting
copley/motor_msg wing
copley/motor_msg sting